lib/mintacoin/accounts/crypto/stellar.ex

```diff
@@ -6,45 +6,89 @@ defmodule Mintacoin.Accounts.Stellar do
   alias Mintacoin.Accounts.Crypto.AccountResponse
   alias Stellar.{Horizon, Horizon.Transaction, KeyPair, TxBuild}
 
-  @type status :: :ok | :error
-  @type stellar_response :: map()
+  @type account :: TxBuild.CreateAccount.t()
```

Comment:
@Odraxs you can use an alias for this type and other similar implementations:

```suggestion
   alias Stellar.TxBuild.{CreateAccount}

  @type account :: CreateAccount.t()
```

Checkout this [documentation about multiple alias](https://elixir-lang.org/getting-started/alias-require-and-import.html#multi-aliasimportrequireuse).